docs/releasenotes/11.1.0.rst

```diff
@@ -0,0 +1,58 @@
+11.1.0
+------
+
+Security
+========
+
+TODO
+^^^^
+
+TODO
+
+:cve:`YYYY-XXXXX`: TODO
+^^^^^^^^^^^^^^^^^^^^^^^
+
+TODO
+
+Backwards Incompatible Changes
+==============================
+
+TODO
+^^^^
+
+Deprecations
+============
+
+TODO
+^^^^
+
+TODO
+
+API Changes
+===========
+
+TODO
+^^^^
+
+TODO
+
+API Additions
+=============
+
+Check for zlib-ng
+^^^^^^^^^^^^^^^^^
+
+You can check if Pillow has been built against the zlib-ng version of the
+zlib library, and what version of zlib-ng is being used::
+
+    from PIL import features
+    features.check_feature("zlib_ng")  # True or False
+    features.version_feature("zlib_ng")  # "2.2.2" for example, or None
+
+Other Changes
+=============
+
+zlib-ng use in wheels
+^^^^^^^^^^^^^^^^^^^^^
+
+Wheels are now built against zlib-ng, for improved speed.
```

Comment:
Can we quantify it? Perhaps something like "more than twice as fast for higher compression levels".